build.rs

```diff
@@ -15,3 +21,29 @@ fn manage_docs() {
 
 #[cfg(not(any(feature = "embed-lgpl-docs", feature = "purge-lgpl-docs")))]
 fn manage_docs() {}
+
+#[cfg(feature = "test")]
+fn generate_tests() {
```

Comment:
I don't think cfg(test) nor this actually works in build.rs ...

Reply:
It doesn't, known cargo problem

Reply:
So, what can we do about this? Do we have to generate the gschema even for normal builds, or should we generate it in Settings' test module?

Reply:
Part of the test I guess